dpsim-villas/include/dpsim-villas/InterfaceSampleBased.h

```diff
@@ -59,7 +59,8 @@ namespace DPsim {
 				for (auto attr : intf.mImportAttrs) {
 					mModifiedAttributes.push_back(attr);
 				}
-				mModifiedAttributes.push_back(Scheduler::external);
+				//TODO: Is this necessary / what effect does a dependency on external have?
```

Comment:
Main reason for indicating the external dependency is to stop the scheduler from dropping the task.

Reply:
I think, as long as there are other tasks who depend on the attributes in mModifiedAttributes, the task should be scheduled either way (I just tried that and it seems to work). If there are no other tasks which depend on this tasks, not scheduling it should actually be the proper choice. For this specific use case it should not matter though, so we can just leave the dependency on external in.